ConnectorGrasshopper/ConnectorGrasshopper/Ops/Operations.VariableInputReceiveComponent.cs

```diff
@@ -424,8 +425,20 @@ private async Task ResetApiClient(StreamWrapper wrapper)
       try
       {
         ApiClient?.Dispose();
-        var acc = await wrapper.GetAccount();
-        ApiClient = new Client(acc);
+        Account account = null;
+        try
+        {
+          account = wrapper?.GetAccount().Result;
+        }
+        catch (Exception e)
+        {
+          AddRuntimeMessage(GH_RuntimeMessageLevel.Warning, e.ToFormattedString());
+          account = new Account
```

Comment:
Here's the "fake account". It basically just contains the  **server url** and the **stream id**.